scripts/sysman/engine.yml

```diff
@@ -121,6 +134,7 @@ desc: "Get the activity stats for an engine group"
 class: $sEngine
 name: GetActivity
 details:
+    - "This function also returns the engine activity inside a Virtual Machine (VM), in the presence of hardware virtualization (SRIOV)"
```

Comment:
i think this line was already added in earlier commit #134 . so we could remove this. not sure why it isn't flagged as merge conflict